REVIEWING.md

```diff
@@ -1,10 +1,12 @@
 # Reviewing pull requests
 
-Thanks to GitHub Actions, an artifact (=zip file) of the rendered protocol (= version with layout applied in html output format) is now automatically created on each pull request.
-
 Reviewers can proceed like this:
 
-1.  On the PR page, you can find a "details" link under "checks - On PR, build the site and ...". Go there, click on the top link in the left sidebar, and download the generated artifact at the bottom of the page.
-2.  Unzip it (to any location you like)
-3.  Go to the unzipped folder location and navigate to the `index.html` file (= rendered version of the protocol) and open with your browser
-4.  [Review the protocol](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request) with the aid of the rendered version.
+1.  To preview the rendered version:
+
+    ```         
+    1.   If you have cloned the protocolsource repo, checkout the branch you need to review and use `protocolhelper::render_protocol("<enter the protocolcode here>")` to render the protocol
+    2.   Alternatively, ask the author of the protocol to do this for you. The author can then upload a zip-archive of the rendered protocol to the conversation pane of the pull request.
+    ```
```

Comment:
Kleine suggestie: misschien moet je (evt. mondeling aan de protocolschrijvers) ook even de suggestie geven om het via drive ofzo even door te geven?  Afhankelijk van wie de protocolschrijver en reviewer zijn, kan de hier beschreven procedure toch al een hele stap zijn (ervaring met html verdeeld over meerdere bestanden, zippen,...).  Anderzijds is het niet minder complex dan dat eerdere artefact, dus voor je geschreven tekst en de meeste gebruikers volstaat dit wel, denk ik.